Resources/Prototypes/Catalog/Fills/Items/belt.yml

```
@@ -141,6 +141,11 @@
     - id: HydroponicsToolMiniHoe
     - id: HydroponicsToolSpade
     - id: RobustHarvestChemistryBottle
+    - id: HydroponicsToolClippers #floof
+    - id: HydroponicsToolHatchet #floof
+    - id: EZNutrientChemistryBottle #floof
+    - id: PlantBGoneSpray #floof
```

Best to group those together
```suggestion
    - id: HydroponicsToolClippers #floof
    - id: HydroponicsToolHatchet #floof
    - id: RobustHarvestChemistryBottle
    - id: EZNutrientChemistryBottle #floof
    - id: PlantBGoneSpray #floof
```